core/src/test/java/fr/sncf/osrd/api/InfraLoadingTest.java

```diff
@@ -1,33 +1,51 @@
 package fr.sncf.osrd.api;
 
-import static fr.sncf.osrd.utils.takes.TakesUtils.readBodyResponse;
+import static fr.sncf.osrd.api.PathfindingTest.contains400;
+import static fr.sncf.osrd.utils.takes.TakesUtils.readHeadResponse;
 import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Configure your code editor so that you never have to `import ...*`, i.e. everything, from a library mate ;)